.tool-versions

```diff
@@ -1,2 +1,2 @@
-elixir 1.15
-erlang 26.1.2
+elixir 1.16.2-otp-26
+erlang 26.2.5
```

Comment:
I wonder if there is a way to make it less restrictive?

Reply:
The point of .tools-versions is to get everyone who's working on this directly using the same version - it's deliberately highly prescriptive so that, for example, we're not all using different versions to run `mix format` :)